xdsl/dialects/llvm.py

```diff
@@ -610,6 +610,83 @@ def verify(self, verify_nested_ops: bool = True):
         super().verify(verify_nested_ops)
 
 
+class ICmpPredicateFlag(StrEnum):
+    EQ = "eq"
+    NE = "ne"
+    SLT = "slt"
+    SLE = "sle"
+    SGT = "sgt"
+    SGE = "sge"
+    ULT = "ult"
+    ULE = "ule"
+    UGT = "ugt"
+    UGE = "uge"
+
+
+@irdl_attr_definition
+class ICmpPredicateAttr(EnumAttribute[ICmpPredicateFlag]):
```

Comment:
I believe you want `OpaqueSyntaxAttribute` or `SpacedOpaqueSyntaxAttribute` here to get it to print properly